snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -421,11 +421,6 @@ def _download_snap(self, build: Dict[str, Any]) -> None:
         for url in urls:
             snap_name = _get_url_basename(url)
```

Comment:
side comment, this should probably be snap_file_name

Reply:
`_get_url_basename()` is used to get the last component in the URL. I would agree if this was the only use of this function.  However, `_get_url_basename()` is also used to get the build # from the URL.  With that context, feel free to suggest a better name :+1:

Reply:
scratch my prior stupid comment, I misunderstood :man_facepalming: 
fixing

Reply:
fixed